index.bs

```diff
@@ -2870,6 +2871,8 @@ value and terminate the operation.
         DOMString                                               rpId;
         sequence<PublicKeyCredentialDescriptorJSON>             allowCredentials = [];
         DOMString                                               userVerification = "preferred";
+        DOMString                                               attestation = "none";
+        sequence<DOMString>                                     attestationFormats = [];
```

Comment:
I think attestationFormats is also missing from PublicKeyCredentialCreationOptionsJSON.